examples/tree-view/containers/Node.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { Component } from 'react'
+import { connect } from 'react-redux'
+import * as actions from '../actions'
```

Comment:
Since this is an exemple, I don't think this syntax is helping readability. Later you can't determine where the actions are coming from.